data_handling/models/topics.models.js

```diff
@@ -1,10 +1,26 @@
 const db = require('../../db/connection');
 
-
-
 exports.fetchTopics = () => {
     return db.query(`SELECT * FROM topics`).then((result) => {
         return result.rows;
     })
 };
 
+exports.fetchArticles = (id) => {
+    let query = `SELECT * FROM articles`;
+
+    if (id) {
+        query += ` WHERE article_id = ${id};`;
+    }
+
```

Comment:
As much as I like this solution I would use a separate controller and model for article_id. One of the later tasks will ask you to get articles with queries which will cause this model to become quite big and complex already